Functions/Functions.csproj

```diff
@@ -23,36 +22,26 @@
 		<PackageIcon>icon.png</PackageIcon>
 		<PackageReadmeFile>README.md</PackageReadmeFile>
 		<RepositoryUrl>https://github.com/supabase-community/functions-csharp</RepositoryUrl>
-	</PropertyGroup>
-	<PropertyGroup>
 		<IncludeSymbols>true</IncludeSymbols>
 		<SymbolPackageFormat>snupkg</SymbolPackageFormat>
-	</PropertyGroup>
-	<PropertyGroup>
 		<GenerateDocumentationFile>true</GenerateDocumentationFile>
-	</PropertyGroup>
-	<PropertyGroup>
 		<Nullable>enable</Nullable>
 		<LangVersion>8.0</LangVersion>
 		<WarningsAsErrors>CS8600;CS8602;CS8603</WarningsAsErrors>
 	</PropertyGroup>
+
 	<PropertyGroup Condition=" '$(Version)' == '' ">
 		<VersionPrefix Condition=" '$(VersionPrefix)' == '' ">1.3.1</VersionPrefix>
-		<VersionSuffix Condition=" '$(VersionSuffix)' == '' ">

```

Comment:
Unfortunately this is a problem, the version is supplied here so that it can be added to the request headers here (for Supabase's internal analytics AFAIK): 

https://github.com/supabase-community/functions-csharp/blob/165ad390c75ca94c6f205977890d7e157503f5a4/Functions/Client.cs#L114C18-L114C18

I have to reject this PR until it is added back in.

Reply:
Ooops. I spoke to soon and didn't read the `Suffix` over the `Prefix`. Ignore me!